public/client.js

```diff
@@ -1,16 +1,13 @@
+//control variable to check whether game is in-progress
+let playing = false;
```

Comment:
also is this for checking if the game is on to prevent other people from joining?

Reply:
Oh technically that could work like that if it was part of the game.. maybe we can then make it part of the game object? Here it was just to not register onkeyUp and onkeydown when you enter the username

Reply:
yeah that would be a nice idea so I can reuse it